app-text/htmltidy/tidy-5.6.0.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="The granddaddy of HTML tools, with support for modern standards"
+DESCRIPTION="Tidy tidies HTML and XML. It can tidy your documents by itself, \
+and developers can easily integrate its features into even more powerful tools."
+HOMEPAGE="http://www.html-tidy.org/"
+COPYRIGHT="1998-2015 World Wide Web Consortium"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/htacg/tidy-html5/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="08a63bba3d9e7618d1570b4ecd6a7daa83c8e18a41c82455b6308bc11fe34958"
+SOURCE_DIR="tidy-html5-$portVersion"
+PATCHES="tidy-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
```

Comment:
3 cases:

1) the libs build on x86_gcc2 primary arch, but the `cmd:tidy` does not.
In this case you should not have the `cmd:tidy` in PROVIDES when building for x86_gcc2 primary arch.

2) both the libs and the cmd build (and work) fine on x86_gcc2 primary arch.
In this case there is no need for commandSuffix and commandBinDir. You can simply drop $binDir when building for a secondary arch.

3) the recipe does not build on x86_gcc2.
In this case you would have to mark this arch as broken.
(But I guess it builds)

Reply:
3, forgot to mark it as broken, created an issue for this on htmltidy github ... will fix :)